src/workers/newNotificationV2Mail.ts

```diff
@@ -296,6 +295,24 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       ),
     };
   },
+  squad_featured: async (con, _, notification) => {
+    const source = await con
+      .getRepository(Source)
+      .findOneBy({ id: notification.referenceId });
```

Comment:
Not important since it's bg, but could add select to only do name and image.

Reply:
Yeah, every bit counts. Let's do that.